### PR TITLE
Only instanton3.0

### PR DESCRIPTION
**Pull request by litman90**
Please see PR #118

### CODE REVIEW

File: tools/py/Instanton_postproc.py [hunk 1]
```diff
@@ -127,6 +128,11 @@
     raise ValueError(
         "We can not indentify the case. The valid cases are: 'reactant', 'TS' and 'instanton'"
     )
+if asr not in list(["poly", "crystal", "none"]):
```
Comment (mahrossi) [inline tools/py/Instanton_postproc.py:131]: Do we issue any warnings if a molecule is linear and one is trying to use the "poly" mode? That is a rather general question but could be somewhat addressed also here and would impact the calculation of the instanton rates. As far as I remember, our "poly" mode assumes the molecule is _not_ linear.

**Comment by mahrossi** [comment]
Looks like the CI is hanging though?

**Comment by litman90** [comment]
> Looks like the CI is hanging though?
The merge of master into this branch generated the hanging problem. Let me check it.

File: ipi/engine/motion/instanton.py [hunk 2]
```diff
@@ -1422,10 +1502,31 @@ def initialize(self, step):
         self.update_old_pos_for()
         self.init = True
 
+    def post_step(self, step, activearrays):
```
[the remark this answers is not in the document]
Comment (litman90) [reply ipi/engine/motion/instanton.py:1510]: ok

File: ipi/utils/instools.py [hunk 3]
```diff
@@ -285,3 +288,14 @@ def print_instanton_hess(prefix, step, hessian, output_maker):
     outfile = output_maker.get_output(prefix + ".hess_" + str(step), "w")
     np.savetxt(outfile, hessian.reshape(1, hessian.size))
     outfile.close_stream()
+
+
+def ms_pathway(pos, m3):
```
[the remark this answers is not in the document]
Comment (litman90) [reply ipi/utils/instools.py:294]: ok

File: ipi/utils/instools.py [hunk 4]
```diff
@@ -3,6 +3,9 @@
 from ipi.utils.messages import verbosity, info
 from ipi.utils import units
 import ipi.utils.mathtools as mt
+import os.path
+import glob
+import copy
 
 
 def banded_hessian(h, im, masses=True, shift=0.001):
```
[the remark this answers is not in the document]
Comment (litman90) [reply ipi/utils/instools.py:11]: ok

File: tools/py/Instanton_postproc.py (hunk 1 above)
Comment (litman90) [reply tools/py/Instanton_postproc.py]: I added linear as a valid option for the asr variable.

**Comment by litman90** [comment]
> > Looks like the CI is hanging though?
> > The merge of master into this branch generated the hanging problem. Let me check it.

After merging PR #122 into master this problem was fixed

File: ipi_tests/regression_tests/instanton/Non_uniform_rate/init.xyz [hunk 5]
```diff
@@ -0,0 +1,8 @@
+6
```
[the remark this answers is not in the document]
Comment (litman90) [reply ipi_tests/regression_tests/instanton/Non_uniform_rate/init.xyz]: ok

File: tools/py/Instanton_postproc.py (hunk 1 above)
Comment (mahrossi) [reply tools/py/Instanton_postproc.py]: 👍

**Review by mahrossi — APPROVED** [review]
Works for me!

**Pull request merged**